eth/fetcher/tx_fetcher.go

```diff
@@ -260,20 +259,9 @@ func (f *TxFetcher) loop() {
 			// Send out all block header requests
```

Comment:
not `block header`, should be `transaction`
```suggestion
			// Send out all transaction requests
```